src/gConsent/manage/getAccessGrantAll.ts

```diff
@@ -38,17 +40,21 @@ import { getSessionFetch } from "../../common/util/getSessionFetch";
 import type { BaseGrantBody } from "../type/AccessVerifiableCredential";
 import { isAccessGrant } from "../guard/isAccessGrant";
 import { isBaseAccessGrantVerifiableCredential } from "../guard/isBaseAccessGrantVerifiableCredential";
-import type { AccessGrant } from "../type/AccessGrant";
 import { getInherit, getResources } from "../../common/getters";
 import { normalizeAccessGrant } from "./approveAccessRequest";
 
 export type AccessParameters = Partial<
   Pick<IssueAccessRequestParameters, "access" | "purpose"> & {
     requestor: string;
     resource: URL | UrlString;
+    status?: ("granted" | "denied")[];
```

Comment:
Is it necessary to make this an array? Since there are only two options possible, and this is for filtering, you either want one of these values ("only the granted ones") or you don't filter on it and don't provide a value at all. 

This is an issue with backwards compatibility though, since we want the current behavior to remain unchanged. I think we can work around this by introducing a third shorthand, `all`, which would behave the same way as an `undefined` value in the eventual API.